CONTRIBUTING.md

```diff
@@ -186,37 +191,42 @@ See how a minor change to your commit message style can make you a better progra
 
 Format: `<type>(<scope>): <subject>`
 
-`<scope>` is optional
+`<scope>` is optional. If your change is specific to one/two packages, consider adding the scope. Scopes should be brief but recognizable, e.g. `content-docs`, `theme-classic`, `core`
 
-## Example
+The various types of commits:
```

Comment:
worth mentioning that it's mostly for maintainers when squashing commits, we don't care too much about internal PR commits